exercises/practice/knapsack/.meta/src/Knapsack.example.elm

```diff
@@ -0,0 +1,71 @@
+module Knapsack exposing (Item, maximumValue)
+
+
+type alias Item =
+    { value : Int
+    , weight : Int
+    }
+
+
+maximumValue : Int -> List Item -> Int
+maximumValue capacity items =
+    doMaximumValue items (List.repeat (capacity + 1) 0)
+
+
+last : List Int -> Int
+last list =
+    List.foldl (\x _ -> x) 0 list
+
+
+valueAt : List a -> Int -> Result String a
+valueAt list index =
+    case list of
+        [] ->
+            Err "Index out of bounds"
+
+        n :: rem ->
+            if index == 0 then
```

Comment:
This doesn't really matter, but in this situation I would generally do
```suggestion
            if index <= 0 then
```
to avoid infinite recursion if someone enters a negative index (which I know won't happen here)